lib/paranoia/version.rb

```diff
@@ -1,3 +1,3 @@
 module Paranoia
-  VERSION = '3.0.1'.freeze
+  VERSION = '3.0.2'.freeze
```

Comment:
Removing Rails 6.1 and Ruby 2.7 and 3.0 are breaking changes. Updating a patch version seems too small

Reply:
good call